core/src/main/java/org/infinispan/interceptors/locking/NonTransactionalLockingInterceptor.java

```diff
@@ -40,6 +41,11 @@ protected Object visitDataWriteCommand(InvocationContext ctx, DataWriteCommand c
       return visitNonTxDataWriteCommand(ctx, command);
    }
 
+   @Override
+   public Object visitRemoveExpiredCommand(InvocationContext ctx, RemoveExpiredCommand command) throws Throwable {
+      return super.visitRemoveExpiredCommand(ctx, command);
```

Comment:
Is this really needed?

Reply:
No :) Was just there to add a debug break point and forgot to remove.